app/src/main/java/fr/free/nrw/commons/category/CategoryItem.java

```diff
@@ -3,6 +3,10 @@
 import android.os.Parcel;
 import android.os.Parcelable;
 
+/**
```

Comment:
Are the java doc changes in this class related to the PR? If not it would be great if you could create another PR for it